no-computer.md

```diff
@@ -1,43 +1,51 @@
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+A language meets their needs because it allows the professor to tell all of his students exactly what to do, have them understand exactly what needs to be done, and do it in a specific way that is understandable to the professor, grader, and other students.
+
 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
+It excites me because I do matjh problem sets almost every day and dont know how i would learn the material without them. Also, without problem sets I wouldnt be able to get specific help so it is really relevant to me. I came up with it when I was using latex to write a problem set.
 
 
 ### Domain
 _Describe the project's domain in five words._
+Students and professors of quantitatives.
 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
-
+The user in this case is the professor who writes the program (the PS) that is then executed by the computer (the student). Programming looks like making up problems or selecting them from a problem book and assigning them to the students. If you dont do it in this way, you really arent using the language at all!
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
+When a program runs a student does the problems in the program and explains how he did them. A program interacts with the user by being graded and errors might occur if the student misunderstood the problem, made a computational answer, could not solve the problem, or was lazy. This is communicated in the output of the program but is not overtly identified as an error. Bad error messages in this lanugage!!
 
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
+It should be easy to regulate the work that students are doing, it would be possible to make qualitative homework assignments using this language but not as easy and it would be seriously impossible to try and do anything non-homework-related in this language.
```

Comment:
What are the verbs and nouns in this language? How does putting the problems in different orders yield a different result? It seems more like a list of problems than a language.